terraform-ecs/ecs.tf

```diff
@@ -0,0 +1,255 @@
+resource "aws_ecs_cluster" "sso_ecs_cluster" {
+  name = "loki-cluster"
+}
+
+resource "aws_ecs_task_definition" "loki_write" {
+  family                   = "loki-task"
+  execution_role_arn       = aws_iam_role.loki_execution_role.arn
+  task_role_arn            = aws_iam_role.loki_task_role.arn
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = "256"
+  memory                   = "512"
+
+  container_definitions = jsonencode([{
+    name      = "loki-write"
+    image     = "ghcr.io/bcgov/sso-grafana:ssoteam-2010"
```

Comment:
@jlangy could you update the image name and version? we need each for sandbox and prod